providers/azureblob/src/main/java/org/jclouds/azureblob/config/InsufficientAccessRightsException.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+ 
+/**
+ * Handles the inability of SAS Authentication string to authenticate the getAcl and setAcl requests.
+ * 
+ */
+ 
+package org.jclouds.azureblob.config;
+
+import com.google.common.util.concurrent.UncheckedExecutionException;
+
+public class InsufficientAccessRightsException extends UncheckedExecutionException {
```

Comment:
I think this should just extend Exception or RuntimeException, depending on whether you want to make this checked or not.

Reply:
implemented both: extending RuntimeException in the 'staging' branch and Exception in the 'test' branch